test/lib.logger.js

```diff
@@ -4,21 +4,23 @@ var logger = require('../lib/logger');
 
 before(function() {
   var winston = require('winston');
-  logger.add(winston.transports.Memory);
+  logger.add(new winston.transports.Console({
+    silent: true
+  }));
 });
 
 it('should be able to log messages of valid type', function(done) {
-  logger.once('logging', function (transport, level, msg, meta) {
-    var isValid = 'error' === level && -1 !== msg.indexOf('test') && 'bar' === meta.foo;
+  logger.once('data', function (transport) {
```

Comment:
Suggest renaming the parameters to something like `info` since I think it's no longer a transport.